Jakefile

```diff
@@ -161,17 +161,30 @@ function generateChromeManifest() {
   });
 }
 
+/**
+ * Exports the files to a target, used to export into mozilla-central
+ *
+ */
+function exportFiles(target) {
+  let mcLocation = getMozillaCentralLocation();
+  let extTargetDir = path.join(mcLocation, target);
+  jake.rmRf(extTargetDir);
+  jake.cpR(BUILD_DIR, extTargetDir);
+
+  console.log(`Exported built sources into ${extTargetDir}`);
+}
+
 desc(`Builds the extension into the ${BUILD_DIR}/ directory`);
 task("build", ["building:cleanup", "building:copy"], () => {});
 
 desc("Exports the sources into mozilla-central");
 task("export-mc", ["build"], () => {
-  let mcLocation = getMozillaCentralLocation();
-  let extTargetDir = path.join(mcLocation, "browser/extensions/webcompat");
-  jake.rmRf(extTargetDir);
-  jake.cpR(BUILD_DIR, extTargetDir);
+  exportFiles("browser/extensions/webcompat");
+});
 
-  console.log(`Exported built sources into ${extTargetDir}`);
+desc("Exports the sources into the mozilla-central for android");
+task("export-mc-android", ["build"], () => {
```

Comment:
Also, as an addition: updates for this extension can and will be served over addons.mozilla.org and the updated via the auto updater. Having different addon contents (like, for example, modifying `ua_overrides.jsm` in a preprocessor), would mean we'd actually end up having two independent extensions, `webcompat-desktop@mozilla.org` and `webcompat-android@mozilla.org`, which means that, for every update, we'd need two QA signoffs and duplicating a lot of other work. I don't see how that's better than copying the exact same source into two folders in mozilla-central.

I feel like I am missing your point here. Mind hopping on Vidyo tomorrow to have a quick discussion on that? :)